infra/promtail/promtail-config.yaml

```diff
@@ -0,0 +1,26 @@
+server:
```

Comment:
It would be helpful to include some links to where we can learn more about promtail and its configuration. I know Google is my friend, but still if you have any good sources to highlight, we could put them here.

Reply:
Yeah, I added one in the PR description, but can leave it here as well!